src/components/AppView/AppView.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+
+import { IDeployment } from "../../shared/types";
+import AppHeader from "./AppHeader";
+import "./AppView.css";
+
+interface IAppViewProps {
+  namespace: string;
+  releasename: string;
+  chartname: string;
+  deployment: IDeployment;
+  getDeployment: (namespace: string, deployname: string) => Promise<{}>;
+}
+
+class AppView extends React.Component<IAppViewProps> {
+  public componentDidMount() {
+    const { namespace, releasename, chartname, getDeployment } = this.props;
+    const deployname = releasename + `-` + chartname;
```

Comment:
Guessing the name isn't always going to work, what if there are multiple deployments (e.g. wordpress and mariadb)? A better way would be to parse the Deployment objects from the `release.manifest` string. I think we can just extract the kinds & names from `release.manifest` and then fetch the deployments from the API to get the latest data. We'll also want this view to display other resources (e.g. services), so we'll be able to reuse the parsing for that.